packages/fxa-auth-server/lib/geocoding.ts

```diff
@@ -0,0 +1,75 @@
+import {
+  Client,
+  PlaceType2,
+  Status,
+} from '@googlemaps/google-maps-services-js';
+import countries from 'i18n-iso-countries';
+import { ConfigType } from '../config';
+import { Logger } from 'mozlog';
+const config: ConfigType = require('../config').getProperties();
+
+export type ZipLocation = {
+  city: string;
+  country: string;
+  countryCode: string;
+  state: string;
+  stateCode: string;
+};
+
+/**
+ *
+ * @param zip
+ * @param country - Two-letter country code ([ISO 3166-1 alpha-2] or corresponding Country name
+ * @returns
+ */
+export async function getLocationFromZip(
+  log: Logger,
+  zip: string,
+  country: string = 'US'
+): Promise<ZipLocation | {}> {
+  const client = new Client();
```

Comment:
If the `getLocationFromZip` function was a method on a class, the Client could be created once on instantiation. That typically helps most SDK implementations as they'll leave HTTP keep-alives intact, or for Google SDK's that use gRPC, it leaves the connection established.